app/controllers/DigitalPack.scala

```diff
@@ -26,11 +29,16 @@ class DigitalPack extends Controller with CommonActions {
     }
   }
 
-  def landingPage(code: String) = NoCacheAction { _ =>
+  def landingPage(code: String) = NoCacheAction.async { _ =>
     val digitalEdition = getById(code).getOrElse(INT)
-    val plan = TouchpointBackend.Normal.catalogService.unsafeCatalog.digipack.month
-    val price = plan.charges.price.getPrice(digitalEdition.countryGroup.currency).getOrElse(plan.charges.gbpPrice)
-    Ok(views.html.digitalpack.info(digitalEdition, price))
+    touchpointBackend.catalogService.catalog.map(_.map { catalog =>
+      val plan = catalog.digipack.month
+      val price = plan.charges.price.getPrice(digitalEdition.countryGroup.currency).getOrElse(plan.charges.gbpPrice)
+      Ok(views.html.digitalpack.info(digitalEdition, price))
+    }.valueOr { err =>
+      logger.error(s"failed landingPage: ${err.list.mkString(", ")}")
```

Comment:
"Failed to load the Digital Pack landing page..." ?

Reply:
why not